scripts/src/release/releasechecker.py

```diff
@@ -94,9 +132,11 @@ def check_if_only_version_file_is_modified(api_url):
 
     return version_file_found
 
-def check_if_release_branch(sender,pr_branch,pr_body,api_url):
+def check_if_dev_release_branch(sender,pr_branch,pr_body,api_url,pr_head_repo):
 
-    if not sender==os.environ.get("BOT_NAME"):
+    print("[INFO] check if PR is release branch on dev")
+
+    if not sender==os.environ.get("BOT_NAME") and not sender==DEFAULT_BOT_NAME:
```

Comment:
My main question is is this enough to avoid a hacker from trying to create an auto-mergable PR, branch name pattern is not enough, I think it comes down to this check, who can impersonate our bot?

Reply:
I added an extra check to make sure the user is also in the OWNERS file. 

I think this is enough, the exposure would be if they could change the BOT_NAME secret in the charts repo or log on as the openshift-helm-charts-bot, which if they could do either we would be in deep trouble.

Reply:
Note release flow fails because the build is running the build scripts from main but he scripts from the pull request:

```
AttributeError: 'NoneType' object has no attribute 'endswith'
Error: Process completed with exit code 1.
```

Not sure we can avoid that.